Error running prototype in `/prototypes/basic`

When I go into `/prototypes/basic`, run `npm i` and then `npm run dev`, I get:

```
Error: Cannot find module 'multer'
Require stack:
- /Users/simon/data-import/lib/importer/index.js
- /Users/simon/data-import/prototypes/basic/app/routes.js
- /Users/simon/data-import/prototypes/basic/node_modules/govuk-prototype-kit/lib/utils/index.js
- /Users/simon/data-import/prototypes/basic/node_modules/govuk-prototype-kit/server.js
- /Users/simon/data-import/prototypes/basic/node_modules/govuk-prototype-kit/listen-on-port.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/Users/simon/data-import/lib/importer/index.js:1:16)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
```

--- a/Makefile
+++ b/Makefile
@@ -5,6 +5,7 @@ prototype:
 ifndef NAME
     $(error Need a value for NAME, e.g., make prototype NAME=value)
 endif
+	@cd lib/importer && npm install
 	@mkdir -p prototypes/${NAME}
 	@cd prototypes/${NAME} && npx govuk-prototype-kit@latest create && npm install ../../lib/importer
 	@echo "\

